src/components/alerts/LrAlertDialog.vue

```diff
@@ -15,74 +15,76 @@
 * please email opensource@seagate.com.
 -->
 <template>
-  <v-dialog v-model="dialog" scrollable max-width="600px">
-    <v-card>
-      <v-card-title>
-        <div class="title-container">
-          {{modalTitle}}
-          <img
-            :src="require(`@/assets/icons/close-green.svg`)"
-            @click="dialog = false"
-            class="close-btn"
-            alt="logo"
-          />
-        </div>
-      </v-card-title>
-      <v-divider></v-divider>
-      <v-card-text style="height: 300px;">
-        <v-treeview v-if="moreDetails.length" :items="moreDetails[0].children"></v-treeview>
-      </v-card-text>
-    </v-card>
-  </v-dialog>
+  <div>
+    <v-dialog v-model="dialog" scrollable max-width="600px">
+      <v-card>
+        <v-card-title>
+          <div class="title-container">
+            {{modalTitle}}
+            <img
+              :src="require(`@/assets/icons/close-green.svg`)"
+              @click="dialog = false"
+              class="close-btn"
+              alt="logo"
+            />
+          </div>
+        </v-card-title>
+        <v-divider></v-divider>
+        <v-card-text style="height: 300px;">
+          <v-treeview v-if="moreDetails.length" :items="moreDetails[0].children"></v-treeview>
+        </v-card-text>
+      </v-card>
+    </v-dialog>
+  </div>
 </template>
 <script lang="ts">
 import { Component, Vue, Prop, PropSync } from "vue-property-decorator";
 import LrLabel from "../shared/LrLabel/LrLabel.vue";
 @Component({
-  name: "LrAlertDialog",
-  components: {},
+	name: "LrAlertDialog",
```

Comment:
Have you changed the tab space configuration for prettier?

Reply:
Tab size is 2 only in my vscode

Reply:
Fine. Anyway, we need to have a discussion in setting up the rules for the formatter.